jaxrs-api/src/main/java/jakarta/ws/rs/core/Context.java

```diff
@@ -23,7 +23,11 @@
 import java.lang.annotation.Target;
 
 /**
- * This annotation is used to inject information into a class field, bean property or method parameter.
+ * <p>This annotation is used to inject information into a class field, bean property or method parameter.</p>
+ *
+ * <p><Note that future versions of this API will stop supporting injection via
```

Comment:
```suggestion
 * <p>Note that future versions of this API will stop supporting injection via
```

Looks like there is an extra `<` in the text.

Reply:
Fixed